app.js

```diff
@@ -0,0 +1,19 @@
+const mobileMenuIcon = document.getElementById('mobile_menu_icon');
```

Comment:
- [ ] Kindly make sure that you have the element in your body that has this `id` as a selector. At this moment the `mobile_menu_icon` selector can not be found anywhere in your HTML file which is cause the mobile menu to open once we click on the hamburger icon.